### DISCUSSION TITLE
How do you output Github PR links?

### DISCUSSION THREAD

**Discussion by esemeniuc** [lindell/multi-gitter#246] [Q&A]
After `multi-getter run ...`, I see output like 
```
Repositories with a successful run:
  mything/sender #89
  mything/listener #133
  mything/borges #156
  mything/delphi #103
```

It would be very nice to have a way to output direct urls that point to github :)

**Comment by lindell** [comment]
What is your use for this? Do you need to pipe it to another command or simply want to open the PR link?

If the later, the link should be on the text already.
![image](https://user-images.githubusercontent.com/284868/162558349-310c8251-63e1-4a03-9439-4eb0d3e77424.png)

**Reply by esemeniuc** [reply]
Thanks for the quick reply! I'm mainly looking to copy the output to share on slack to have the PRs reviewed. On my terminal (Intellij terminal), there are no hyperlinks when clicking on outputs. Perhaps there is a terminal feature that is not well known? What terminal do you use?

**Reply by lindell** [reply]
It does work in all terminal emulators I've tried it on (except CMD on windows) which includes:
- iTerm2
- Standard Mac terminal (I think)
- Windows terminal
- Hyper

I'm happy to include a plain text flag that can print the link instead of putting it on the text if a PR is made for it 🙂

**Reply by esemeniuc** [reply]
I just tried with Alacritty on Mac OS, and I also do not have any option to click the link. Cmd+click, Option+click, or Ctrl+click all do not do anything. An option for plain text would be super nice :)

**Reply by hansbogert** [reply]
Running into this as well, @lindell  did you imply you would make the change, or that you are open to PRs addressing this?
